anu-001/FirstForFood
# #160641620 Build Out Authentication Endpoints for Admin and Users

#### What does this PR do?
* Have both POST/ method for sign up and login for protected endpoints 
* Have dummy class for order and menu controller to test the routes and lay out the project struture

#### Description of Task to be completed?
Have the following endpoints working
* POST  /api/v2/auth/signup
* POST  /api/v2/auth/login

#### How should this be manually tested?
Clone the repoisitory and change directory in to it, run npm install
* Using Postman test every endpoints above with this header
key: Content-Type: value : application/x-www-form-urlencoded
* Signup with an username, email, password, phone, address and  get your token, for testing.
* To use your token, prefix the generated token with 'token' and space then the jwt value
* To access admin page ensure you register your username as 'adminuser', to have access to admin protected routes 
* To access the user routes, register with any parameters 
       
#### Any background context you want to provide?
All inputs from client are validated

#### What are the relevant pivotal tracker stories?
[#160641620](https://www.pivotaltracker.com/story/show/160641620)
